src/state_machine.zig

```diff
@@ -127,19 +118,19 @@ pub const StateMachine = struct {
         input: []u8,
     ) void {
         // Guard against the wall clock going backwards by taking the max with timestamps issued:
-        self.prepare_timestamp = std.math.max(
-            // The cluster `commit_timestamp` may be ahead of our `prepare_timestamp` because this
+        self.pending_timestamp = std.math.max(
+            // The cluster `post_timestamp` may be ahead of our `pending_timestamp` because this
             // may be our first prepare as a recently elected leader:
-            std.math.max(self.prepare_timestamp, self.commit_timestamp) + 1,
+            std.math.max(self.pending_timestamp, self.post_timestamp) + 1,
             @intCast(u64, realtime),
         );
-        assert(self.prepare_timestamp > self.commit_timestamp);
+        assert(self.pending_timestamp > self.post_timestamp);
```

Comment:
i.e. They don't relate to two-phase commit transfers. This is also part of the motivation for this PR, so that we don't overload these consensus terms for two-phase payments.

Reply:
Yeah, I wondered about that. Thank you for the clarification. :-)